Doc/library/datetime.rst

```diff
@@ -295,6 +295,20 @@ Instance attributes (read-only):
 
    Between 0 and 86,399 inclusive.
 
+   .. warning::
```

Comment:
Consider using `caution` instead (thanks to Alex who discovered this directive while we were working on https://docs.python.org/3.14/reference/datamodel.html#type.__annotations__ ).